circle.yml

```diff
@@ -27,10 +25,28 @@ jobs:
     steps:
       - checkout
       - restore_cache:
-          key: node-deps-{{ checksum "package-lock.json" }}
+          key: node-deps-{{ checksum "yarn.lock" }}
+      - run:
+          name: greenkeeper-lockfile
+          command: yarn global add greenkeeper-lockfile@1
+      - run:
+          name: Add Yarn global bin to path
+          command: echo 'export PATH=$PATH:`yarn global bin`' >> $BASH_ENV
+      - run:
+          name: greenkeeper-lockfile-update
+          command: greenkeeper-lockfile-update
+      - run:
+          command: yarn --pure-lockfile
```

Comment:
you shouldn't need to run yarn again here since you are restoring the node modules from cache up above